src/components/landing/index.js

```diff
@@ -118,193 +121,80 @@ export class Landing extends Component {
       <div className="Landing" onClick={this.hideTooltips}>
         <div className="LandingHero">
           <div className="content">
-            <h2>September 8th - 10th, 2017</h2>
+            <h2>It happened September 8th - 10th, 2017</h2>
             <h1>Create, code, and<br />learn with us in Porto</h1>
-            <h3>Join us in the playground of the future for 3 days</h3>
-
-            <Link to="/signup" className="signup">
-              <Button straight cta large onClick={this.scrollToTop}>Apply Now</Button>
-            </Link>
-            <span className="open-until">Registration is open until September 2nd</span>
+            <h3>For 3 days you've joined join us in the playground for the future</h3>
 
-            <nav className="LandingSections">
-              <ul>
-                <li className="hackathon"><a href="#hackathon">Hackathon</a></li>
-                <li className="workshops"><a href="#workshops">Workshops</a></li>
-                <li className="schedule"><a href="#schedule">Schedule</a></li>
-              </ul>
-            </nav>
+            <a href="#join-slack" className="signup">
+              <Button straight large inactive>Apply Now</Button>
```

Comment:
This button should also be `disabled`. If that interferes with styling in a way that can't be cleanly solved, then clicks on `Button[inactive]` elements should be a ignored